dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormWithToggle.vue

```diff
@@ -12,7 +12,7 @@
     </h5>
   </div>
   <div v-show="dataPointIsAvailable">
-    <div class="form-field" v-if="dataPointIsAvailable">
+    <div class="form-field">
```

Comment:
Why was the if removed. I see that the v-show above handled the visibility but is there maybe a difference in the data send, now, if the datapoint is disabled?

Reply:
Because we want the fields to be sent as an empty string if the category is disabled. At least I think so.

Reply:
I would prefer sending a null object instead of empty strings for each field. You could clarify that in the group chat if you insist on this.

Reply:
changed